### PR TITLE
Add standard git flow image

### PR DESCRIPTION
**Pull request by Nhacsam**

### CODE REVIEW

File: git/git_1.md [hunk 1]
```diff
@@ -177,4 +177,37 @@ $ git log --graph --oneline
...
+When you work alone on a side project, you often don't need to use the branch system. Working in the master branch is enought.
+
+Things begin to be harder as soon as you are working on a web application which is in production. Imagine the following scenario : 
+1. You made change in your codebase and push it in the server
+2. You begin the following feature
+3. A critical bug occured on the server
+
+You cannot fix the bug without sending your incomplete works on the new feature in the server. The solution to that problem is to have, a least, a separate branch for each server you have. The branch must be always synchronized with the server to always be able to accept hotfixes.
+
+To solve all that kind of problem, there is a standard git flow used in most of the projects. This is a way to organize your branch in a team. The schema of the flow is visible in the figure below.
```
Comment (Almouro) [inline git/git_1.md]: To solve all that kind of problem -> j'ai pas compris désolé :[

File: git/git_1.md [hunk 2]
```diff
@@ -177,4 +177,37 @@ $ git log --graph --oneline
 * 5d77249 Initial commit
 ```
 
+The two commit was seperated in two diferent timeline. Now a new commit have been created to merge everything in the master branch.
```
Comment (yleflour) [inline git/git_1.md]: The two commits were separated into two...

Comment (yleflour) [inline git/git_1.md]: Now a new commit has been created to merge everything into the master branch

File: git/git_1.md [hunk 3]
```diff
@@ -177,4 +177,37 @@ $ git log --graph --oneline
...
+
+
+### The standard git flow
+
+When you work alone on a side project, you often don't need to use the branch system. Working in the master branch is enought.
```
Comment (yleflour) [inline git/git_1.md]: *enough

File: git/git_1.md [hunk 4]
```diff
@@ -177,4 +177,37 @@ $ git log --graph --oneline
...
+
+Things begin to be harder as soon as you are working on a web application which is in production. Imagine the following scenario :
```
Comment (yleflour) [inline git/git_1.md]: Things start to get harder...

File: git/git_1.md [hunk 5]
```diff
@@ -177,4 +177,37 @@ $ git log --graph --oneline
...
 
+The two commit was seperated in two diferent timeline. Now a new commit have been created to merge everything in the master branch.
+
+
+### The standard git flow
+
+When you work alone on a side project, you often don't need to use the branch system. Working in the master branch is enought.
+
+Things begin to be harder as soon as you are working on a web application which is in production. Imagine the following scenario : 
+1. You made change in your codebase and push it in the server
```
Comment (yleflour) [inline git/git_1.md]: You made a change in your codebase and pushed it

**Comment by yleflour** [comment]
@Nhacsam Il faut merger celle d'Alexandre avant de faire une 2nde review

File: git/git_1.md [hunk 6]
```diff
@@ -177,4 +177,37 @@ $ git log --graph --oneline
...
+2. You begin the following feature
```
Comment (yleflour) [inline git/git_1.md]: You start to work on the next feature

File: git/git_1.md [hunk 7]
```diff
@@ -177,4 +177,37 @@ $ git log --graph --oneline
...
+3. A critical bug occured on the server
+
+You cannot fix the bug without sending your incomplete works on the new feature in the server. The solution to that problem is to have, a least, a separate branch for each server you have. The branch must be always synchronized with the server to always be able to accept hotfixes.
```
Comment (yleflour) [inline git/git_1.md]: You cannot fix the bug without sending your incomplete work on the new feature to the server. The solution to that problem is to have, a least, one separate branch for each server you have. These branch must always be synchronized with their respectful server to always be able to accept hotfixes

**Pull request merged**